packages/replay-core/src/t.ts

```diff
@@ -5,11 +5,34 @@ export type TextureFont = {
   /**
    * Font name
    */
-  name: string;
+  family?: string;
   /**
    * Size of font in relation to game size
    */
-  size: number;
+  size?: number;
+  /**
+   * Font weight, either a string like `"bold"` or a number like `500`
+   */
+  weight?: number | string;
+  /**
+   * Font style, typically either `"normal"` or `"italic"`
+   *
+   * @default "normal"
+   */
+  style?: string;
```

Comment:
I liked the unions you had before, it helps with autocomplete... We can pair for a bit to get the types sorted out.